c7n/resources/vpc.py

```diff
@@ -1272,6 +1292,35 @@ def process(self, resources):
                 method = getattr(client, 'revoke_security_group_%s' % label)
                 method(GroupId=r['GroupId'], IpPermissions=groups)
 
+            if self.data.get('PermissionType'):
+                p_type = self.data.get('PermissionType')
```

Comment:
afaics none of the add functionality is covered by tests in the latest pr?

Reply:
I might need some help on adding test cases